.github/workflows/main.yml

```diff
@@ -4,7 +4,7 @@ jobs:
   test:
     strategy:
       matrix:
-        go-version: [1.13.x, 1.14.x, 1.15.x]
+        go-version: [1.15.x, 1.16.x]
         os: [ubuntu-latest]
     runs-on: ${{ matrix.os }}
     env:
```

Comment:
Hi @devimc , should we switch `GO111MODULE` from `off` to `auto` here so with 1.16 it is built as module-aware?

Reply:
@wainersm thanks, change applied